cmd/server/assets/login/register-phone.html

```diff
@@ -100,8 +107,62 @@
       let $submitPin = $('#sms-code-submit');
       let $resendPin = $('#sms-code-resend');
 
+      let $registeredDiv = $('#registered-div');
+      let $factors = $('#factors');
+
       let verId = ""
 
+      firebase.auth().onAuthStateChanged(function(user) {
+        if (!user) {
+          return
+        }
+
+        if (user.multiFactor.enrolledFactors.length > 0) {
+          $skip.text("Account settings");
+          $skip.attr("href","/account");
+
+          for (i = 0; i < user.multiFactor.enrolledFactors.length; i++) {
+            let factor = user.multiFactor.enrolledFactors[i];
+            let li = $('<li class="list-group-item"></li>');
+            li.attr("id", "factor"+factor.uid)
+
+            let row = $('<div></div>').text(factor.displayName);
+            li.append(row)
+
+            let unenroll = $('<a href="#" class="text-danger float-right" title="Delete this factor"></a>');
+            unenroll.tooltip();
+            unenroll.append('<span class="oi oi-delete mr-1" aria-hidden="true"></span>');
+            row.append(unenroll)
+
+            let time = $('<small class="row text-muted ml-1"></small>').text('Enrolled at: ' + factor.enrollmentTime);
+            row.append(time);
+
+            unenroll.on('click', function(event) {
+              unenrollFactor(factor);
+            });
+
+            $factors.append(li);
```

Comment:
This is going to append the `<li>` to the parent `<div>` with an id of #factors. Did you mean to put the ID on the `<ul>`?